charts/am-org-role-mapping-service/values.preview.template.yaml

```diff
@@ -77,7 +77,8 @@ orm:
 
 am-role-assignment-service:
   java:
-    image: 'hmctspublic.azurecr.io/am/role-assignment-service:latest'
+    # image: 'hmctspublic.azurecr.io/am/role-assignment-service:latest'
+    image: 'hmctspublic.azurecr.io/am/role-assignment-service:pr-2487'
```

Comment:
This temporary override must be reversed before merger: but only after merger of hmcts/am-role-assignment-service#2487.